dev-db/percona-xtrabackup/percona-xtrabackup-2.4.13.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Would 7 work here?

Reply:
Oops, it builds fine, but it fails to run:

````
# ldd /usr/bin/xtrabackup 
        linux-vdso.so.1 (0x00007ffecd956000)
        libpthread.so.0 => /lib64/libpthread.so.0 (0x00007f68e74a3000)
        libgcrypt.so.20 => /usr/lib64/libgcrypt.so.20 (0x00007f68e7186000)
        libcrc.so => not found
...
````